turbo/templatetags/turbo_streams.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.core.signing import Signer
+
+from turbo import channel_name_for_instance
+
+register = template.Library()
+
+# https://docs.djangoproject.com/en/3.1/topics/signing/
+
+
+@register.inclusion_tag('turbo/turbo_stream_source.html')
+def turbo_stream_from(model_instance):
+    signer = Signer()
+    channel_name = channel_name_for_instance(model_instance)
+    signed_channel_name = signer.sign(channel_name)
+    return {"signed_channel_name": signed_channel_name}
+
+
+@register.inclusion_tag('turbo/turbo_stream_id.html')
+def stream_id(model_instance):
+    model_name = model_instance._meta.verbose_name.lower()
```

Comment:
The Django docs define `verbose_name `as "A human-readable name for the object", so i'm not sure it's the best identifier we could find. Why not use the model name ?

Reply:
I think the verbose name is generally the model name unless it's overridden –– this ensures we're keeping with the "rails-y" conventions for names, and it'll also be predictable (the same name that shows up in the admin)